Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:22-alpine AS modules_dev
+FROM node:20-alpine AS modules_dev
```

```suggestion
FROM node:20.17-alpine AS modules_dev
```

Probably ought to pin to .17 in all of these, too

---

Thanks. I'm also looking into whether we want an engines key because satisfying or lowering sentry/profiling-node may need to be looked at